Cargo.toml

```diff
@@ -15,7 +15,7 @@ license = "MIT"
 
 [dev-dependencies]
 criterion = "0.4.0"
-compiletest_rs = "0.9.0"
+#compiletest_rs = "0.10.0"
```

Comment:
What's up with the rename / comment out of the compile_tests?

Reply:
uncommented